files/aws/lib/systemd/system/systemd-networkd.service.d/aws-override.conf

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
it looks like this file is present twice.. `20-override.conf` and `aws-override.conf`.. is this intentional?

Reply:
Thanks for pointing this. 20-override.conf has been removed. I think I had deleted it earlier but looks like my steps were not correct.